equinix/fabric_mapping_helper.go

```diff
@@ -4,7 +4,6 @@ import (
 	"fmt"
 	equinix_schema "github.com/equinix/terraform-provider-equinix/internal/fabric/schema"
 	"log"
-
```

Comment:
You could leave this line in. No need to delete and the sections in imports makes it easier to read.

Reply:
I think you should just `git reset` this file to wherever you started it from. There was no need to make any changes to it. Good `git` practice to figure out how to get this returned to the beginning while keeping other changes.